src/components/DAO/DAO.tsx

```diff
@@ -1,22 +1,540 @@
+    try {
+      const membersData = await fetch(
+        "https://api.github.com/orgs/stabilitydao/public_members"
+      );
+      const dataJson = await membersData.json();
+      setMembers(dataJson);
+      console.log(dataJson);
+    } catch (error) {
+      console.log("Fetching getTeamData", error);
+    }
+  };
+
+  const fetchPlatformData = async () => {
+    if ($publicClient && $balances) {
+      try {
+        const platformVersion: any = await $publicClient.readContract({
+          address: platform,
+          abi: PlatformABI,
+          functionName: "platformVersion",
+        });
+
+        const platformFees: any = await $publicClient.readContract({
+          address: platform,
+          abi: PlatformABI,
+          functionName: "getFees",
+        });
+
+        const contractData: any = await $publicClient.readContract({
+          address: platform,
+          abi: PlatformABI,
+          functionName: "getData",
+        });
+
+        const profitTotalSupply = await $publicClient.readContract({
+          address: "0x48469a0481254d5945E7E56c1Eb9861429c02f44",
+          abi: ERC20ABI,
+          functionName: "totalSupply",
+        });
+
+        const _pmTotalSupply = await $publicClient.readContract({
+          address: "0xAA3e3709C79a133e56C17a7ded87802adF23083B",
+          abi: IERC721Enumerable,
+          functionName: "totalSupply",
+        });
+
+        const _sdivTotalSupply = await $publicClient.readContract({
+          address: "0x9844a1c30462B55cd383A2C06f90BB4171f9D4bB",
+          abi: ERC20ABI,
+          functionName: "totalSupply",
+        });
+
+        const treasuryBalance = await $publicClient.readContract({
+          address: "0x48469a0481254d5945E7E56c1Eb9861429c02f44",
+          abi: ERC20ABI,
+          functionName: "balanceOf",
+          args: ["0xC82676D6025bbA6Df3585d2450EF6D0eE9b8607E"],
+        });
+
+        const multisigBalance = await $publicClient.readContract({
+          address: "0x48469a0481254d5945E7E56c1Eb9861429c02f44",
+          abi: ERC20ABI,
+          functionName: "balanceOf",
+          args: [contractData[0][6]],
+        });
+
+        //tvl
+        const totalTvl: bigint = $vaults[6].reduce(
+          (total: bigint, numero: bigint) => total + numero,
+          BigInt(0)
+        );
+        const _totalTvl = formatUnits(totalTvl, 18);
+
+        //profit Token
+        const _profitTotalSupply = formatUnits(profitTotalSupply, 18);
+        const _profitTokenPrice = Number(totalTvl) / Number(profitTotalSupply);
+        const profitMarketCap =
+          Number(_profitTokenPrice) * Number(_profitTotalSupply);
+
+        const profitToken: ProfitTokenData = {
+          price: _profitTokenPrice,
+          totalSupply: _profitTotalSupply,
+          marketCap: profitMarketCap,
+        };
+
+        setProfitTokenData(profitToken);
+
+        //sdiv token
+        setSdivTotalSupply(formatUnits(_sdivTotalSupply, 18));
```

Comment:
const [prTotalSupply, setPmTotalSupply] = useState("");
  const [sdivTotalSupply, setSdivTotalSupply] = useState(""); 
  
  ////////
  const [tokensTotalSupply,setTokensTotalSupply] = useState({pm:"",sdiv:""})
  
  to many useStates, if you have 2 or more tokens with the same data types, it’s better to use 1 useState, this applies to everything, not just this particular case